games-emulation/pcsx2/pcsx2-9999.ebuild

```diff
@@ -42,9 +42,15 @@ DEPEND="${RDEPEND}
 
 pkg_setup() {
 	if [[ ${MERGE_TYPE} != binary && $(tc-getCC) == *gcc* ]]; then
-		if [[ $(gcc-major-version) -lt 4 || $(gcc-major-version) == 4 && $(gcc-minor-version) -lt 8 ]] ; then
```

Comment:
please keep changes limited to the topic of git summary

Reply:
I splitted commits

Reply:
Rather drop the whole check - all GCC versions up to 5.5.0 are masked at this point, thus not supported.

Reply:
It's typo - look one line up. There is no whitespace.  
Since there is (masked) versions of gcc in portage I think this check is useful.  
However if you insist I will drop this check - one space just doesn't worth time we spent on it.